pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -309,8 +313,8 @@ public static AggregationFunction getAggregationFunction(FunctionContext functio
           case DISTINCTAVGMV:
             return new DistinctAvgMVAggregationFunction(firstArgument);
           case DISTINCT:
-            return new DistinctAggregationFunction(arguments, queryContext.getOrderByExpressions(),
```

Comment:
@Jackie-Jiang  can we clean this up so that Distinct is not modeled as an aggregation function?